File: cgi-bin/ruby/hello.rb
#!/usr/bin/ruby -w

colors = Array["aqua", "black", "blue", "fuchsia", "gray", "green", "lime", "maroon", "navy", "olive", "purple", "red", "silver", "teal", "white", "yellow"];

time = Time.new;


print "Content-type: text/html \n\n";
print "<!doctype html>";
print "<html>\n<head>\n<title>Hello Ruby World!!!!</title>\n</head>\n";
print "<body style=background-color:"+ colors.sample  + ">\n";
print "<h1>Hello World from Ruby @ " + time.strftime("%Y-%M-%D | %H:%M:%S")+ "</h1>\n";
print "</body>\n</html>";

File: cgi-bin/py/hello.py
#!/usr/bin/python

from random import randrange
from datetime import datetime
from pytz import timezone

colors = ["aqua", "black", "blue", "fuchsia",
        "gray", "green", "lime", "maroon",
        "navy", "olive", "purple", "red",
        "silver", "teal", "white", "yellow"]

fmt = "%Y-%m-%d %H:%M:%S"
time = datetime.now(timezone('America/Los_Angeles'))


print "Content-type: text/html \n\n";
print "<!doctype html>"
print ""
print "<html>"
print "<head><title>Hello Python Based CGI World!</title></head>"
print "<body style='background-color: " + colors[randrange(len(colors))] + "'>"
print "<h1>Hello World from Python @ " + time.strftime(fmt) + "</h1>"
print "</body>"
print "</html>"
